onnxruntime/core/framework/tensorprotoutils.h

```diff
@@ -76,6 +76,13 @@ ONNXTensorElementDataType GetTensorElementType(const ONNX_NAMESPACE::TensorProto
 template <size_t alignment>
 common::Status GetSizeInBytesFromTensorProto(const ONNX_NAMESPACE::TensorProto& tensor_proto, size_t* out);
 
+/**
+Special marker used to indicate an existing memory buffer contains the TensorProto external data.
+If the 'location' field of the external data info is set to this marker, the 'offset' field should contain the
+address of the memory containing the data.
+*/
+constexpr const ORTCHAR_T* kMemoryAddressTag = ORT_TSTR("*/_ORT_MEM_ADDR_/*");
```

Comment:
nit: could the name be more specific since its in the general `onnxruntime::utils` namespace? e.g., `kTensorProtoMemoryAddressTag`